src/sections/layout/footer/Footer.tsx

```diff
@@ -12,14 +12,14 @@ interface FooterProps {
 export function Footer({ dataverseInfoRepository }: FooterProps) {
   const { t } = useTranslation('footer')
   const { dataverseVersion } = useDataverseVersion(dataverseInfoRepository)
-
+  const currentYear = new Date().getFullYear().toString()
```

Comment:
Could you add an empty line before the return of the jsx here? to improve readability 👍🏼

Reply:
Yes, thanks for pointing out this!